test_util/config.ts

```diff
@@ -36,13 +31,13 @@ class TestConfig extends Config<string, CallMethods, SubscriptionMethods, ErrorD
   }
 }
 
-function requiresSetupError(): never {
-  throw new Error([
-    "You are using a config that depends on the `test_ctx` util.",
-    "You may want to try re-running the same command, but prefixed with the run command for `test_ctx`.",
-    "For instance, `deno task run examples/metadata.ts` would (in this repository)",
-    "become `deno task test_ctx deno task run examples/metadata.ts`.",
-  ].join(" "));
+async function testCtx(): Promise<never> {
+  const testCtxPath = new URL("../test_ctx.ts", import.meta.url).pathname;
+  const p = Deno.run({
+    cmd: ["deno", "task", "run", testCtxPath, "deno", "task", "run", Deno.mainModule],
+  });
+  await p.status();
+  Deno.exit(1);
```

Comment:
I believe the child process status should be forwarded

```suggestion
  const { code } = await p.status();
  Deno.exit(code);
```